examples/wdio/mocha/mocha.test.js

```diff
@@ -1,6 +1,4 @@
 describe('webdriver.io page', () => {
-    it('should be a pending test')
```

Comment:
can we revert this change? It has nothing to do with the actual issue.

Reply:
Done.